app/boards/arm/planck/planck_rev6.dts

```diff
@@ -44,6 +44,20 @@
             ;
     };
 
+    left_encoder: left_encoder {
+        compatible = "alps,ec11";
+        a-gpios = <&gpiob 12 GPIO_PULL_UP>;
+        b-gpios = <&gpiob 13 GPIO_PULL_UP>;
+        steps = <80>;
+        status = "okay";
```

Comment:
There did not appear to be a clear precedent for setting the status of an encoder on a unibody board. Most boards set this to "disabled" in the dts and then set the respective encoder statuses to "okay" in the overlays for each half. Let me know if there is a better way to handle this.

Reply:
Ideally you'd have it disabled, with a commented out section in the *keymap* to enable the encoder node if you're using it.

Reply:
> Ideally you'd have it disabled, with a commented out section in the _keymap_ to enable the encoder node if you're using it.

Cool. Will do.

Reply:
Encoder is now disabled by default with commented-out node in keymap to enable it.